target/dita/index.ditamap

```diff
@@ -9,7 +9,7 @@
     <topicref href="./regions/regions.dita" format="dita">
         <topicmeta/>
     </topicref>
-    <topicref href="./regions/britain/britain_complex/unit_daf.dita" format="dita">
+    <topicref href="./regions/britain1/britain_complex/unit_daf.dita" format="dita">
```

Comment:
in our _generated_ DITA code I'm happy to use the real country name.  This issue is just about how we read in the data.